Add empty io check to data integrity and repair

anvit opened the pull request:
Add ability to search for and delete information objects with all columns set to NULL in the data integrity and repair task.

anvit committed:
Add empty io check to data integrity and repair

Add ability to search for and delete information objects with all
columns set to NULL in the data integrity and repair task.

melaniekung reviewed (approved):
Thanks @anvit, lots of users will be happy with this addition !

anvit committed:
Add ability to check invalid IO in data integrity

Add the ability to check/delete descriptions that only have a slug and
object table entry but no information_object entries.

anvit committed:
Add logging for empty column data integrity case

melaniekung reviewed (approved):
Thanks @anvit , LGTM!
